### DISCUSSION TITLE
CU-UP / CU-CP deployment

### DISCUSSION THREAD

**Discussion by tomwassing** [srsran/srsRAN_Project#251] [General Help] [answered]
Hi,

As mentioned in the roadmap, the CU/DU split deployment is on the horizon. Could you provide an estimated timeline for this update? Additionally, I'm curious to know if there are plans for further disaggregation with the CU-UP/CU-CP deployment.

Kind regards,

Tom

**Comment by brendan-mcauliffe** [comment, marked as answer]
Hi @tomwassing, the CU/DU split is expected to be released Q2 2024. The CU-CP/ CU-UP split is planned, but a timeline for this is not available yet.
